### PR TITLE
Feature/orca 536 Add checks to create_release.sh

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Addresses [ORCA-536: Add checks to create_release.sh](https://bugs.earthdata.nasa.gov/browse/ORCA-536)

## Changes

*added checks to bin/create_release.sh
* fixed bug in bin/build_release_package.sh

## Type of change

- [x ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
tested doing a release both locally and via bamboo without error
https://ci.earthdata.nasa.gov/browse/ORCA-OI286-RO-5/

## Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas

- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
    - [x ] Manual tests (outlined above)
- [x ] My code has passed security scanning
    - [x ] Snyk
    - [ x] git-secrets

### CODE REVIEW

File: bin/create_release.sh [hunk 1]
```diff
@@ -44,8 +44,16 @@ if [[ ! $bamboo_RELEASE_FLAG == true ]]; then
   exit 0
 fi
 
+## Check if the code has already been released.
+export url="https://github.com/nasa/cumulus-orca/releases/tag/"v$bamboo_ORCA_VERSION""
+if curl --output /null --silent --fail "$url"; then
+  echo "Release URL already exists: $url. Exiting."
+  exit 0
```
Comment (AndrewDorn) [inline bin/create_release.sh:51]: Good check. I do lean towards having this be an `exit 1` to make the failure a bit noisier.

File: bin/create_release.sh [hunk 2]
```diff
@@ -60,8 +68,14 @@ export RELEASE_URL=$(curl \
     | sed -e 's/.*\(https.*\)\"\,/\1/' \
     | sed -e 's/api/uploads/')
 
-# Release Package
-echo $RELEASE_URL
+# Release URL is created only if there is valid github token
+if [[ ! $RELEASE_URL ]]; then
+  echo "RELEASE_URL is empty. Exiting"
```
Comment (AndrewDorn) [inline bin/create_release.sh:73]: Just a bit more info here. Something like "This may be caused by an invalid 'GITHUB_TOKEN'."

**Review by AndrewDorn — CHANGES_REQUESTED** [review]
Looks good to me. A couple style updates, plus one question about the curl return code.

File: bin/create_release.sh (hunk 1 above)
Comment (rizbihassan) [reply bin/create_release.sh:51]: ok fair enough.

File: bin/create_release.sh (hunk 2 above)
Comment (rizbihassan) [reply bin/create_release.sh:73]: added

File: bin/create_release.sh [hunk 3]
```diff
@@ -60,8 +68,14 @@ export RELEASE_URL=$(curl \
     | sed -e 's/.*\(https.*\)\"\,/\1/' \
     | sed -e 's/api/uploads/')
```
[the remark this answers is not in the document]
Comment (rizbihassan) [reply bin/create_release.sh:69]: looks like it does not catch the error after testing

**Review by bhazuka — APPROVED** [review]

**Review by AndrewDorn — APPROVED** [review]

**Pull request merged**